src/TwigComponent/src/MountedComponent.php

```diff
@@ -53,6 +53,11 @@ public function getAttributes(): ComponentAttributes
         return $this->attributes;
     }
 
+    public function setAttributes(ComponentAttributes $attributes): void
```

Comment:
Do you by any chance have an alternate solution? (don't want to waste your time)

MountedComponent was "almost" immutable and i don't fear your usage right now at all..  
but it may be seen as an open door 

I'm just asking by chance... but not a big deal :)

Reply:
I was looking for another way, but I didn't find anything :/. This solutions is a result of my talk with Ryan 😅.

Reply:
Well let's ignore my comment then :)

Reply:
This can be removed now, right?